api/external/data_feed/data_feed.proto

```diff
@@ -0,0 +1,136 @@
+syntax="proto3";
+
+package chef.automate.api.datafeed;
+option go_package = "github.com/chef/automate/api/external/data_feed";
+
+import "google/api/annotations.proto";
+
+// for option (chef.automate.api.policy)
+import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
+// for option (chef.automate.api.iam.policy)
+import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
+
+service DatafeedService {
+    rpc AddDestination(Destination) returns (DestinationResponse){
+		option (google.api.http) = {
+			post: "/datafeed/destination"
+			body: "*"
+		};
+		option (chef.automate.api.policy) = {
+			resource: "datafeed:destination"
+			action: "add"
+		};
+		option (chef.automate.api.iam.policy) = {
+			resource: "datafeed:destination"
+			action: "datafeed:destination:add"
+		};
+	};
+    rpc GetDestination(DestinationId) returns (Destination) {
+		option (google.api.http) = {
+			get: "/datafeed/destination/{id}"
+		};
+		option (chef.automate.api.policy) = {
+			resource: "datafeed:destination:{id}"
+			action: "read"
+		};
+		option (chef.automate.api.iam.policy) = {
+			resource: "datafeed:destination:{id}"
+			action: "datafeed:destination:get"
+		};
+	};
+    rpc DeleteDestination(DestinationId) returns (DestinationResponse){
+		option (google.api.http) = {
+			delete: "/datafeed/destination/{id}"
+		};
+		option (chef.automate.api.policy) = {
+			resource: "destination:destination:{id}"
+			action: "delete"
+		};
+		option (chef.automate.api.iam.policy) = {
+			resource: "destination:destination:{id}"
+			action: "destination:destination:delete"
+		};
+	};
+    rpc UpdateDestination(Destination) returns (DestinationResponse) {
+    		option (google.api.http) = {
+			patch: "/datafeed/destination/{id}"
+			body: "*"
+		};
+		option (chef.automate.api.policy) = {
+			resource: "datafeed:destination/{id}"
+			action: "update"
+		};
+		option (chef.automate.api.iam.policy) = {
+			resource: "datafeed:destination:{id}"
+			action: "datafeed:destination:update"
+		};
+	};
+    rpc ListDestinations(Empty) returns (ListDestinationResponse){
```

Comment:
Instead of `Empty` for the request parameter, you should create a new parameter for each RPC. This allows for easy additions and changes to the RPC. For example, create a `ListDestinationsRequest` that is empty like below. 
```
message ListDestinationsRequest {}
```

Then later if the RPC needs a filter or any other argument it can be easily added to the `ListDestinationsRequest` object without breaking the API.